src/EDMF_Environment.jl

```diff
@@ -355,12 +362,15 @@ function sgs_quadrature(self::EnvironmentThermodynamics, EnvVar::EnvironmentVari
 
             # update cloudy/dry variables for buoyancy in TKE
             EnvVar.cloud_fraction.values[k] = outer_env[i_cf]
+
             self.qt_dry[k] = outer_env[i_qt_dry]
-            self.th_dry[k] = theta_c(self.Ref.p0_half[k], outer_env[i_T_dry])
+            self.th_dry[k] = TD.dry_pottemp_given_pressure(param_set, outer_env[i_T_dry], self.Ref.p0_half[k])
```

Comment:
I think I'd prefer using thermo constructors, rather than the Thermodynamics-internal methods, here:
```suggestion
            ts = TD.PhaseEquil_pTq(param_set, self.Ref.p0_half[k], outer_env[i_T_dry], TD.PhasePartition(outer_env[i_qt_cld], outer_env[i_ql], 0.0))
            ts_dry = TD.PhaseDry_pT(param_set, self.Ref.p0_half[k], outer_env[i_T_dry])
            self.th_dry[k] = TD.dry_pottemp(ts_dry)
```
And then you can use `ts` below.